runtime/oti/j9nonbuilder.h

```diff
@@ -5480,6 +5480,10 @@ typedef struct J9VMThread {
 #define J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_JAVA_HOOKS 0x200001
 #define J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_INTERNAL_HOOKS 0x200002
 #define J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_PHASE_END 0x200003
+#define J9VMSTATE_CRIU_SUPPORT_CHECKPOINT_FAILED_PHASE_START 0x200004
```

Comment:
Im not sure what this options is for, is it document?

Reply:
It looks like there are other vmstates missing, so I'd prefer to add all them in one pass (if necesarry?)

Reply:
https://github.com/eclipse-openj9/openj9/blob/6b03a382a12a2293d1cb4d884c55899859da516e/doc/diagnostics/VMState.md?plain=1#L54

It helps to find the components quicker with the error code.
It could be done in a separate PR though.